frontend/src/data/routes.tsx

```diff
@@ -17,11 +17,9 @@ export const prodRoutes: Route[] = [
 // TODO: add translation
 export const routes: Route[] = [
   { name: "Demo", link: "/dashboard" },
-  { name: "Emission Estimation", link: "/emission-estimation"},
```

Comment:
I felt this should be accessible only when logged in as well but lmk if my assumption was incorrect

Reply:
No these should be able to be used without authentication. They are explicitly designed to function without it. Pleaser revert :)

Reply:
done